docs/getting_started/start_new_component.rst

```diff
@@ -0,0 +1,403 @@
+How to Start Working on a :ref:`Component <what_is_a_component>`
+################################################################
+
+To support :ref:`component <what_is_a_component>` development, SPL Core provides features to:
+
+- document the component behavior
+- test the component behavior
+- generate component report with full traceability between documentation, code and tests
+- **all** component artifacts are configurable (content will be included/excluded based on configuration)

```

Comment:
follow the grammatic of the bullet points above and rephrase the sentence, like "configure **all** component artifacts [...]